ROADMAP.md

```diff
@@ -15,35 +15,38 @@ To check off items on the list, open `ROADMAP.md` in your editor and place an `x
 
 ## Application
 
-- [ ] toolbar
+- [x] toolbar
   - name of application, global menu, and people search
```

Comment:
Do we have a people search yet?

Reply:
Not yet, but there is a separate item for it at the end of the roadmap. This item is just a description of what is in the toolbar on the current 360 site.